### PR TITLE
Fix to CrossContamination tool

### PR DESCRIPTION
**Pull request by mtekman**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Complete rewrite of the underlying scripts due to some ambiguous mapping with the barcodes as reported by a user.

### CODE REVIEW

File: tools/crosscontamination_barcode_filter/crosscontamination_barcode_filter.xml [hunk 1]
```diff
@@ -35,10 +35,12 @@
                     <add value="&#41;"/> <!-- right parenthesis -->
                 </valid>
             </sanitizer>
-        </macro>        
+        </macro>
     </macros>
     <requirements>
         <requirement type="package" version="2.2.1" >r-ggplot2</requirement>
+        <requirement type="package" version="1.11.6" >r-data.table </requirement>
+        <requirement type="package" version="3.0.0" >libgfortran</requirement>
```
Comment (bgruening) [inline tools/crosscontamination_barcode_filter/crosscontamination_barcode_filter.xml]: this should not be needed

Comment (mtekman) [reply tools/crosscontamination_barcode_filter/crosscontamination_barcode_filter.xml]: surprisingly it was...!

File: tools/crosscontamination_barcode_filter/scripts/contamination_plot.R [hunk 2]
```diff
@@ -146,11 +149,13 @@ contaminationPlot <- function(title, columndata, barcode.data, plate.data, RAW)
     }
 
     plotTheme <- function(){
+        print("HEY")
```
Comment (bgruening) [inline tools/crosscontamination_barcode_filter/scripts/contamination_plot.R]: debug statement...

File: tools/crosscontamination_barcode_filter/scripts/crosscontamination_filter.R [hunk 3]
```diff
@@ -9,7 +9,9 @@ if (length(args) != 1){
 }
 
 source(args[1])
-##source('/tmp/tmpckZpsq/job_working_directory/000/8/tmpywIcp1')
+
+## debug in debug dir
+source('../debug/config.debug')
```
Comment (bgruening) [inline tools/crosscontamination_barcode_filter/scripts/crosscontamination_filter.R]: debug ...

**Comment by mtekman** [comment]
apologies, this is still very much a work in progress and I pushed my unfinished changes only so that I could pull them from another machine

**Pull request merged**